shared/js/background/chrome-events.es6.js

```diff
@@ -647,7 +643,7 @@ function getArgumentsObject (tabId, sender, documentUrl) {
     const site = Object.assign({}, tab?.site || {})
     const referrer = tab?.referrer || ''
 
-    const firstPartyCookiePolicy = tdsStorage.config.features?.trackingCookies.settings.firstPartyCookiePolicy || {
+    const firstPartyCookiePolicy = utils.getFeatureSettings('trackingCookies1p').firstPartyTrackerCookiePolicy || {
```

Comment:
`utils.getFeatureSettings('trackingCookies1p')` can return `undefined` which here would throw an exception instead of using the default value.